pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -224,6 +209,21 @@ def _add_adjoint_transforms(program: TransformProgram) -> None:
     return 0
 
 
+def check_gpu_resources() -> None:
+    """Check the available resources of each Nvidia GPU"""
+    if find_library("custatevec") is None and not imp_util.find_spec("cuquantum"):
+
+        raise ImportError(
+            "custatevec libraries not found. Please pip install the appropriate custatevec library in a virtual environment."
```

Comment:
Should we use the official case in error messages?
```suggestion
            "cuStateVec libraries not found. Please pip install the appropriate cuStateVec library in a virtual environment."
```